software/openvisualizer/SConstruct

```diff
@@ -45,6 +45,8 @@ Targets:
         Options
           --sim         Run in simulator mode with default count of motes.
           --simCount=n  Run in simulator mode with 'n' motes.
+          --importTopo  Run in simulator mode with data imported from a previous
```

Comment:
Could you please rename all `importTopo` to `pathTopo`. Please grep throughout your changes (or on this PR)